arbitrator/jit/src/wavmio.rs

```diff
@@ -278,14 +298,17 @@ fn ready_hostio(env: &mut WasmEnv) -> MaybeEscape {
     let position_within_message = socket::read_u64(stream)?;
     let last_block_hash = socket::read_bytes32(stream)?;
     let last_send_root = socket::read_bytes32(stream)?;
-    let last_hotshot_height = socket::read_u64(stream)?;
-    let validating_hotshot_height = socket::read_u64(stream)?;
+    let validating_height = socket::read_u64(stream)?;
     let hotshot_comm = socket::read_bytes32(stream)?;
+    let l1_block_height = socket::read_u64(stream)?;
+    let hotshot_avail = socket::read_u8(stream)?;
 
-    env.small_globals = [inbox_position, position_within_message, last_hotshot_height];
+    env.small_globals = [inbox_position, position_within_message, validating_height];
     env.large_globals = [last_block_hash, last_send_root];
     env.hotshot_comm_map
-        .insert(validating_hotshot_height, hotshot_comm.0);
+        .insert(validating_height, hotshot_comm.0);
+    env.hotshot_avail_map
+        .insert(l1_block_height, hotshot_avail > 0);
```

Comment:
What about the genesis case where hotshot_height = 0? Or is this fine because we assume that we'll always start sequencing at a time greater than genesis? If its the latter, maybe add a comment.

Reply:
We assue that we'll always start from latest block in hotshot.

Reply:
@ImJeremyHe please add a comment as @nomaxg suggested

Reply:
in this commit 19bd7443 we use the `validated_hotshot_height + 1` and this implies that 0 is no longer the case